### ISSUE TITLE
Kitty + zellij + termusic hang and fails to run or draw anything

### ISSUE THREAD

**Issue by Porkepix** [zellij-org/zellij#2211]
**In Case of Graphical or Performance Issues**

1. Delete the contents of `/tmp/zellij-1000/zellij-log`, ie with `cd /tmp/zellij-1000/` and `rm -fr zellij-log/` (`/tmp/` is `$TMPDIR/` on OSX)
2. Run `zellij --debug`
3. Run `stty size`, copy the result and attach it in the bug report
4. Recreate your issue.
5 Quit Zellij immediately with ctrl-q (your bug should ideally still be visible on screen)

Please attach the files that were created in `/tmp/zellij-1000/zellij-log/` to the extent you are comfortable with.

**Basic information**

`zellij --version`: zellij 0.34.4
`stty size`: 45 208
`uname -av` or `ver`(Windows): Linux T470-clement 6.1.12-arch1-1 #1 SMP PREEMPT_DYNAMIC Tue, 14 Feb 2023 22:08:08 +0000 x86_64 GNU/Linux

List of programs you interact with as, `PROGRAM --version`: output cropped meaningful, for example:
`kitty 0.27.1 created by Kovid Goyal`
`Termusic 0.7.9`

**Further information**
Reproduction steps, noticeable behavior, related issues, etc

I simply open Kitty, open a `zellij` session, and then try to run `termusic`. The command will simply hang and do nothing, nothing will be drawn.
If taking `zellij` out of the equation, it works. On other terminals I tried, it works. If replacing `zellij` by `tmux`, it works.

Here are the logs collecting during reproduction, though they seems pretty empty and useless:

General log: [zellij.log](https://github.com/zellij-org/zellij/files/10858977/zellij.log)
Pane log: [zellij-9.log](https://github.com/zellij-org/zellij/files/10858986/zellij-9.log)

Not that after trying to run `termusic`, I waited a good ten seconds before quitting `zellij`.

**Comment by gauthierc** [comment]
The creator of kitty advises against running tmux in kitty, I guess this is also the case with zellij :
https://sw.kovidgoyal.net/kitty/faq/#i-am-using-tmux-and-have-a-problem

**Comment by Porkepix** [comment]
> The creator of kitty advises against running tmux in kitty, I guess this is also the case with zellij : [sw.kovidgoyal.net/kitty/faq/#i-am-using-tmux-and-have-a-problem](https://sw.kovidgoyal.net/kitty/faq/#i-am-using-tmux-and-have-a-problem)

Eh, that starts well "First, terminal multiplexers are [a bad idea](https://github.com/kovidgoyal/kitty/issues/391#issuecomment-638320745), do not use them, if at all possible."…

Note though that the bug reported here isn't there if you use tmux instead of zellij.

**Comment by gauthierc** [comment]
Yes, it is true. But if the bug comes from a particularity of kitty, I doubt that the author of kitty will fix the problem.
I think it's better to improve Zellij to provide the best features of kitty, rather than trying to make it compatible with it.
But this is just my personal point of view.
